rskj-core/src/main/java/co/rsk/PublicKeyUtils.java

```diff
@@ -0,0 +1,40 @@
+package co.rsk;
+
+import co.rsk.bitcoinj.core.Base58;
+import org.bouncycastle.util.encoders.Hex;
+
+import java.util.Arrays;
+
+/**
+ * This class contains methods for converting an Extended Public Key To The Compressed Format
+ *
+ * @author Kelvin Isievwore
+ */
+public class PublicKeyUtils {
+
+    public static String convertXpubToCompressedPublicKey(String xpub) {
+
+        // Decode from Base58Check
+        byte[] decodedXpub = decodeFromBase58Check(xpub);
```

Comment:
```suggestion
        byte[] decodedXpub = Base58.decodeChecked(xpub);
```
We could simply call Base58 class here instead of creating a new function